core-services/ci-secret-bootstrap/README.md

```diff
@@ -0,0 +1,3 @@
+# ci-secret-bootstrap
+
+This directory contains the manifests required to run the [ci-secret-bootstrap](https://github.com/openshift/ci-tools/tree/master/cmd/ci-secret-bootstrap).
```

Comment:
Can you add a sentence about what the tool does, i.e. why we are running it?

Reply:
Updated.